roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -7,6 +7,7 @@
     l_crio_registries: "{{ l2_docker_additional_registries + ['docker.io'] }}"
 - set_fact:
     l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l_crio_registries)) }}"
+    l_is_node_system_container: "{{ (openshift_use_node_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
```

Comment:
There is no need to use set_fact here.

We could do the same exact line in defaults/main.yml